### ISSUE TITLE
Re3gistry 2.6.1 webapp error

### ISSUE THREAD

**Issue by TeijoKal** [ec-jrc/re3gistry#473]
It seems that the webapp cannot show properly if the value of the code is a link.

For example i have value that is a link to the value and the result is the following

<img width="858" alt="error_with_link" src="https://github.com/user-attachments/assets/d7123ac7-bafe-4617-ba8c-650a7d695f65">

So it is just adding the link to the url which does not work.

**Comment by arantzaetxebarria** [comment]
Dear @TeijoKal 

We would like to get more specific information about your problem,
We would appreciate it if you could provide us with some information by asking the following questions:
In which field are you trying to insert the link? 
which type is it? 
is it configured as hyperlink?

Thanks

**Comment by TeijoKal** [comment]
I just added the link like it was also in our old production environment (Re3gistry 1.3.1). We have the registry that there is mixed values. Some are character based values and other are links.

It is inserted in the URI field.

**Comment by arantzaetxebarria** [comment]
Dear @TeijoKal 
Could you send us a screenshot of the administration part to see where the link is placed?

Thanks

**Comment by TeijoKal** [comment]
Here is a screen shot. When adding the value i clicked that it is external value and put the url here

<img width="787" alt="Insert_url_value" src="https://github.com/user-attachments/assets/24424078-e6be-42e1-a11d-f74aafedd29d">

**Comment by TeijoKal** [comment]
And after adding the link the url is uncorrectly showing the following

<img width="760" alt="Insert_url" src="https://github.com/user-attachments/assets/3005f6be-c196-4dad-beab-6c67bdebe64b">

**Comment by arantzaetxebarria** [comment]
Dear @TeijoKal 

I think the URI you entered is not well-formed, isn't it?

![image](https://github.com/user-attachments/assets/965771ba-087a-44b4-a8a8-409d78e1666f)

**Comment by TeijoKal** [comment]
Yes it is well-formed. It was copied from our version 1.3.1 production environment.

And it was inserted in the system as external value and the URI was given with the label. 

The version used to insert was not 2.6.1. The value was inserted already with earlier version 2. But only now it was found out that the value inserted did not work like in our production environment 1.3.1

**Comment by arantzaetxebarria** [comment]
Dear @TeijoKal 

Could you share with us the exact URL you are inserting in the URI field? Many thanks

Best regards

**Comment by arantzaetxebarria** [comment]
The ticket is closed as there has been no response from the user.
@TeijoKal If the problem persists or you have any further questions, please don't hesitate to contact us again. Thanks
